app/jobs/order_cycle_opening_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Trigger jobs for any order cycles that recently opened
+class OrderCycleOpeningJob < ApplicationJob
```

Comment:
`OrderCycleOpenedJob`? Otherwise you could read it as the job that opens the order cycle.

Reply:
Yeah that's a good question. I err-ed on the side of following Matt's precedent with `OrderCycleClosingJob`.

But I think you're right: the order cycle is state considered changed based on time, not the actions performed in these jobs. Some actions are performed, but retrospectively _after_ the (implicit) state change. So would you agree we should also rename it to OrderCycleClos***ed***Job?

Reply:
Agreed, but to rename the existing job in a separate PR. It will require a transition, eg keep the existing named class as a sub-class for compatibility with jobs still in the queue.